src/client/components/measures-tile/measures-tile.tsx

```diff
@@ -103,8 +103,11 @@ export class MeasuresTile extends Component<MeasuresTileProps, MeasuresTileState
 
   private handleGlobalKeyDown = (e: KeyboardEvent) => {
     if (e.shiftKey && e.keyCode === keyCodes.m) {
-      e.preventDefault();
-      this.toggleSearch();
+      const element = e.target as HTMLElement;
```

Comment:
We have similar code for the `dimension` please add it there too (maybe extract the code for it, too) => https://github.com/remerge/turnilo/blob/master/src/client/components/dimension-list-tile/dimension-list-tile.tsx#L178-L183. Something like:

```js
if (originatesFromTextAreaOrInput(e)) return;
```